fixtures/codity_prod_smoke.py

```
@@ -0,0 +1,6 @@
+def run_formula(formula: str, row: dict) -> object:
+    return eval(formula, {}, row)
```

Ruff (0.15.20)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)